e2e/testcases/managed_resources_test.go

```diff
@@ -203,11 +216,28 @@ metadata:
 	}
 }
 
+func writeObjectYAMLFile(path string, obj *unstructured.Unstructured, scheme *runtime.Scheme) error {
+	nsBytes, err := testkubeclient.SerializeObject(obj, ".yaml", scheme)
+	if err != nil {
+		return fmt.Errorf("failed to format yaml: %w", err)
+	}
+	if err := os.WriteFile(path, nsBytes, 0644); err != nil {
+		return fmt.Errorf("failed to create a tmp file: %w", err)
+	}
+	return nil
+}
+
 // TestKubectlCreatesManagedConfigMapResource tests the drift correction regarding kubectl
 // tries to create a managed non-namespace resource in both the mono-repo mode and the multi-repo mode.
 func TestKubectlCreatesManagedConfigMapResource(t *testing.T) {
```

Comment:
One is cluster-scoped (Namespace) and the other is namespace-scoped (ConfigMap). I renamed them for clarity.

But it might be nice in the future to change one to test with RepoSync, not just RootSync.